controllers/awsmachine_controller.go

```diff
@@ -499,6 +498,7 @@ func (r *AWSMachineReconciler) reconcileNormal(_ context.Context, machineScope *
 			conditions.MarkFalse(machineScope.AWSMachine, infrav1.InstanceReadyCondition, infrav1.InstanceProvisionFailedReason, clusterv1.ConditionSeverityError, err.Error())
 			return ctrl.Result{}, err
 		}
+		//return ctrl.Result{RequeueAfter: time.Minute}, nil
```

Comment:
Why was this commented out when updating the tests?

Reply:
Good question. I previously tested this controller with that uncommented - however after reviewing the code and tests today, I don't believe it's necessary, we can just fall through and continue with reconciliation instead of exiting early and re-queuing. I need to confirm it's not behavior we actually need, I'm going to do some more tests with large Machine Deployments tomorrow.

Hence the WIP status.

Reply:
Removed, this was not necessary in my tests today.